src/main/java/httpServer/ExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package httpServer;
+
+public class ExceptionHandler {
```

Comment:
Think this can be a static class with argument passed to `handleException`

Reply:
Or class instance with no state if you need DI

Reply:
Thanks, I'll change this